kqueen_ui/blueprints/ui/views.py

```diff
@@ -508,7 +533,7 @@ def handle(self):
                     if (hasattr(v, 'switchtag') and v.switchtag) and form.provisioner.data in k
                 }
             except Exception as e:
-                self.logger('error', repr(e))
```

Comment:
Did you check what self.logger does exactly? Is the change equivalent?

Reply:
yep, i just moved all log arch to file:
i already defined module naming in logger formatting (logger_config.py), it will be good for:

1) reuse 1 logger and does not create multiple loggers -> increase performance
2) all logging arch stored in one file -> increase visibility
3) possibility of design new logger_config for customers as 1 file-upload

Reply:
self logger just summon created package logger, my structure already include package/module name in logger format, so we can use only 1 non-self logger
all this changes tested, u also can try it by your self, most thing that i afraid of - i may lost smtp during multiple rebases)) but i double check all stuff, and seems all fine